### PR TITLE
chore: v6 doc site updates

### PR DESCRIPTION
**Pull request by micahjones13**
## Brief Description

Adds an alert that notifies users that they are on v6 of astrouxds.
Adds a note about being on v6 on SB.
Updates links on astrouxds to point to v6 where needed
Updates links on SB 
Adds blockquotes to readme's and contributing.md that specifies that this is v6
Removes google analytics

## JIRA Link

https://rocketcom.atlassian.net/browse/ASTRO-4659

## Motivation and Context

If a user has current astrouxds and v6 up in the same browser, they need to be able to tell them apart.

## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.
- [ ] Regressions are passing and/or failures are documented
- [ ] Changes have been checked in evergreen browsers

### CODE REVIEW

**Comment by micahjones13** [comment]
Working on SB v6 notification still

File: packages/astro-uxds/_content/community/propose-a-change.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ If you are new to Git and Git fork workflows the easiest way to edit Astro conte
 
 ## Step 1 - Navigate to the page you want to edit
 
-At the bottom of each page on [AstroUXDS.com](https://astrouxds.com) you will find a link titled **Propose a change or fix to this page**. Follw the link to propose your change on GitHub. If you are not an authorized Astro contributor you will be asked to create a fork of the repository, this is a safe operation. You cannot break Astro proposing a change.
+At the bottom of each page on [AstroUXDS.com](https://astrouxds-v6.com) you will find a link titled **Propose a change or fix to this page**. Follw the link to propose your change on GitHub. If you are not an authorized Astro contributor you will be asked to create a fork of the repository, this is a safe operation. You cannot break Astro proposing a change.
```
Comment (markacianfrani) [inline packages/astro-uxds/_content/community/propose-a-change.md:18]: https://astrouxds-v6.netlify.app/

Comment (markacianfrani) [reply packages/astro-uxds/_content/community/propose-a-change.md:18]: also typo in Follw

**Review by markacianfrani — APPROVED** [review]
few minor typos

**Pull request merged**